cypress/support/hostsfunctions.ts

```diff
@@ -91,19 +91,33 @@ function deleteHostListView(invenotryName: string, host_type: string, hostName:
   cy.contains(/^No results found./);
 }
 
-function deleteHostDetailsView(invenotryName: string, host_type: string, hostName: string) {
+function deleteHostDetailsView(inventoryName: string, host_type: string, hostName: string) {
   // function for delete host
   // can use this for stand alon host and for invntory host
   // will delete and verify that all was deleted curectlly
 
-  navigateToHost(host_type, hostName, '[data-cy="name-column-cell"] a', invenotryName);
+  navigateToHost(host_type, hostName, '[data-cy="name-column-cell"] a', inventoryName);
```

Comment:
just a question - is this line for navigating to a tab? @RedOrNot

Reply:
It seems like that, it is using navigateTo and then clicking and filtering through the GUI.

Reply:
for tabs we have `cy.clickTab(/^Back to OAuth Applications$/, true);`

Reply:
sticking with existing commands helps us understand it better and the changes will be uniform, is it something that you can incorporate?

Reply:
it's not for tab
it's for specific host - navigate to inventory host